### DISCUSSION TITLE
BaramMesh is not starting

### DISCUSSION THREAD

**Discussion by imon00** [nextfoam/baram#95] [General]
Hi Everyone,

I am on windows 11 in a AMD Ryzen 5 laptop. In my system after complete new installation of BARAM, BaramMesh is not starting though I clicked. But BaramFlow is working fine. Did  anyone face the same problem?

https://github.com/user-attachments/assets/7f2002c4-d493-4961-8176-b9bccb71567b

**Comment by intothevoid1986** [comment]
Same behavior but launching BaramFlow, using 24.3.3 for windows.

**Reply by intothevoid1986** [reply]
Solved:
Uninstall application, then remove directory '.BaramFlow' and '.BaramMesh' in c:\users\<my-username> and reinstall again.

**Reply by imon00** [reply]
Thank you.

**Reply by arniendf** [reply]
I ran into the same problem. Thank you. This helped.

**Comment by kdg5424** [comment]
Hello this is Donggyu from NEXTFOAM.

Sorry for the delay. It was thanksgiving day of Korea until yesterday.
And I'm sorry for the inconvenience for using it.

Best regards

Donggyu Kim
